### ISSUE TITLE
Enforce single quote - clarification

### ISSUE THREAD

**Issue by Auric-Manteo** [astral-sh/ruff#7834]
I am trying to get ruff to complain about using double quotes.
In this example I want to have single quotes instead of double quotes: `my_var="text"`
Currently when I run ruff check it does not complain if I use double quotes or single quotes.
If this does not work I don't quite get hat the inline-quotes (I never heard the term  inline quote. Is that "'" an inline quote? :D) and quote-style is for.
Could you clarify?

>  ruff 0.0.292

pyproject.toml 

      [tool.ruff]
      line-length = 120
      
      [tool.ruff.flake8-quotes]
      docstring-quotes = "double"
      inline-quotes = "single"
      
      [tool.ruff.format]
      # Prefer single quotes over double quotes
      quote-style = "single"
      
      [tool.black]
      line-length = 120
      skip-string-normalization = true

**Comment by dhruvmanila** [comment]
Hey, is this related to the linter? If so, you'll have to select the rules explicitly:

```toml
[tool.ruff]
extend-select = ["Q"]
```

The `inline-quotes` settings is for the inline strings which are basically every string apart from the docstrings.

**Comment by Auric-Manteo** [comment]
Fantastic, thank you @dhruvmanila!

**Comment by Borda** [comment]
@dhruvmanila we may have a similar issue; I have set all as suggested:
```toml
[lint]
select = [
    ...
    "Q",
]

flake8-quotes = {inline-quotes = "single"}
```
But still, the single quotes are not enforced, for example [here](https://github.com/OpenDevin/OpenDevin/pull/1425#discussion_r1582056596)
